pmacApp/src/devPmacRam.c

```diff
@@ -1072,7 +1080,11 @@ LOCAL long devPmacRamEvent_read (struct eventRecord *pRec) {
 
   pDpvt = (PMAC_RAM_DPVT *) pRec->dpvt;
 
+#if LT_EPICSBASE(3,15,0,0)
   pRec->val = (short) (0x0000ffff & pDpvt->dpramData.ramLong);
+#else
+  sprintf(pRec->val, "event %d", pDpvt->dpramData.ramLong);
```

Comment:
We probably ought to use epicsSnprintf from epicsStdio.h here instead of sprintf... On the other hand I can see that sprintf has been used throughout tpmac...